interface/portraits/portraits_mantid.gfx

```diff
@@ -12,7 +12,8 @@ spriteTypes = {
 		layer = { # GFX_TYPE:[d|p]INDEX:COLOR_LINK:DONT_REFRESH_IF_VALID:CULTURE_INDEX
```

Comment:
Mantid child portraits not working.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/84382744-22ab9580-abfc-11ea-9514-21521a0e8064.png)

</details>